.github/CODEOWNERS

```diff
@@ -4,3 +4,4 @@ docs/software/communication @msimberg
 docs/software/devtools/linaro @jgphpc
 docs/software/prgenv/linalg.md @finkandreas @msimberg
 docs/software/sciapps/cp2k.md @abussy @RMeli
+docs/software/sciapps/lammps.md @nickjbrowning
```

Comment:
@nickjbrowning you'll need to accept the invite that I just sent you to be part of the cscs-docs group so that github accepts you as a valid code owner.